js/e2e/README.md

```diff
@@ -0,0 +1,30 @@
+## Basic test
+
+To run the test you need to:
+
+```sh
+cd ..
+```
+
+### Start appium
+
+```sh
+npx appium
+```
+
+### Build the app
+
+```sh
+make ios.release
+```
+
+Then grab the app path from the `info Installing` line in the command output, for example:
```

Comment:
Can you explain here that the user will have to close the simulator at this step if it launches?
That `make ios.release` is just the command used to build the app and launch it, but in the case of the test, we are only interested in the build.

Reply:
sure